Tests/TestCombine/FixtureCombine.swift

```diff
@@ -0,0 +1,194 @@
+ import MockoloFramework
+
+ let combineProtocol = """
+/// \(String.mockAnnotation)(subject: dictionaryPublisher = CurrentValueSubject; myPublisher = PassthroughSubject; noDefaultSubjectValue = CurrentValueSubject)
+public protocol Foo: AnyObject {
+    var myPublisher: AnyPublisher<String, Never> { get }
+    var dictionaryPublisher: AnyPublisher<Dictionary<String, String>, Never> { get }
+    var noDefaultSubjectValue: AnyPublisher<CustomType, Never> { get }
+}
+"""
+
+let combineProtocolMock = """
+public class FooMock: Foo {
+    public init() { }
+
+    public var myPublisher: AnyPublisher<String, Never> { return self.myPublisherSubject.eraseToAnyPublisher() }
+    public private(set) var myPublisherSubject = PassthroughSubject<String, Never>()
+
+    public var dictionaryPublisher: AnyPublisher<Dictionary<String, String>, Never> { return self.dictionaryPublisherSubject.eraseToAnyPublisher() }
+    public private(set) var dictionaryPublisherSubject = CurrentValueSubject<Dictionary<String, String>, Never>(Dictionary<String, String>())
+
+    public var noDefaultSubjectValue: AnyPublisher<CustomType, Never> { return self.noDefaultSubjectValueSubject.eraseToAnyPublisher() }
+    public private(set) var noDefaultSubjectValueSubject = PassthroughSubject<CustomType, Never>()
+}
+"""
+
+let combinePublishedProtocol = """
+/// \(String.mockAnnotation)(published: myStringPublisher = myString; myIntPublisher = myInt; myCustomTypePublisher = myCustomType; myNonOptionalPublisher = myNonOptional)
```

Comment:
See the comment on the parameter name. Instead of a user needing to provide a var name to map to, it might be a better approach to autogenerate a unique name, like so:
 
`@mockable(combine: foo = @Published)` 
-> generates `@Published var publishedFoo // or to be more unique, atPublishedFoo, published_Foo, etc`.

This is unless there's a need to map a publisher var to a specific non-publisher var name; if that's the case, how common is it and how are they used in tests?

Reply:
Thanks for the explanation. That makes sense. A follow up question then is, if a publisher var doesn't have a matching non-published var declared in the protocol (or its parent protocols), would it actually need one (how is the publisher fired without it in non-test code?) or would it suffice to just provide a default PassthroughSubject?   

Re: annotation, it could be`@mockable(combine: foo = CurrentValueSubject; barPublisher = @Published bar)` to be more consistent with 'rx'.  Lmk your thoughts.

Reply:
If there's no matching var declared, then maybe it should be treated as an "incorrect" case and we shouldn't generate one at all (PassthroughSubject or not).

Reply:
I updated the implementation to match what we discussed here.

> If there's no matching var declared, then maybe it should be treated as an "incorrect" case and we shouldn't generate one at all (PassthroughSubject or not).

I decided to generate the PassthroughSubject anyway in this case. If we didn't, the user would end up with the typical variable generation. Roughly:

```
///mockable(combine: myIntPublisher = @Published myInt)
protocol Foo {
    var myIntPublisher: AnyPublisher<Int, Never> { get }
}

class FooMock: Foo {
    init() { }
    init(myIntPublisher: AnyPublisher<Int, Never>) {
        self._myIntPublisher = myIntPublisher
    }

    var _myIntPublisher: AnyPublisher<Int, Never>! 

    var myIntPublisher: AnyPublisher<Int, Never> { _myIntPubilsher }
}
```

It seems more developer friendly to always give them a PassthroughSubject at the very least. Less setup on their part.

Reply:
Makes sense, agreed.